Add a paragraph about EE permission application on events (#495)
akeneo/pim-api-docs: content/events-api/more-about-events.md

# More about events

> ## EE permission application

Reviewer: ```suggestion
## Enterprise Edition (EE) permission application
```

::: warning
The following paragraph is for EE customers.
:::

You may ask yourself how EE permissions apply to the event subscription. 

> Indeed, Akeneo PIM uses the permissions related to the connection user group to filter what it has to send or not. 

Reviewer: ```suggestion
Indeed, Akeneo PIM uses the permissions related to the connection user group to filter what it is supposed to send or not. 
```

That's why when you enable the event subscription & setup your connection permissions, you have to keep in mind the following information:
- **Akeneo PIM sends events on products your connection can view.** 

To receive an event, your connection user group must, at least, view one of the product categories.  

> N.b. If the product has no category at all, the connection will receive the event. 

Reviewer: ```suggestion
N.b. By default, if the product is `Unclassified`, the connection will receive the event. 
```

- **Akeneo PIM sends product events even if the updated values aren't accessible by the connection.** 

> For example, Julia updates the ERP code of a product. Your e-commerce connection doesn't have access to the attribute group containing the ERP code but will receive an event anyway.  

Reviewer: ```suggestion
Here are a couple of examples to highlight different scenarios
### Permissions based on attribute groups
The ERP code of a product is updated. Your e-commerce connection doesn't have access to the attribute group containing the ERP code but will receive an event anyway. In the event payload, there will be no mention of the newly updated ERP code, to respect the permission granted at the attribute group level. 
```

> Another example, Julia updates the Italian description of the product. Your e-commerce connection doesn't have access to the Italian locale but will receive an event anyway.  

Reviewer: ```suggestion
### Permissions based on locales
Another example, the translation team updates the Italian description of the product. Your e-commerce connection doesn't have access to the Italian locale but will receive an event anyway. In the event payload, there will be no mention of the newly updated Italian description, to respect the permission granted at the locale level. 
```

> However, Akeneo PIM applies the EE permissions to the data it sends. So, the payload doesn't contain values the connection can't access. 

Reviewer: ```suggestion
```
